bika/health/content/doctor.py

```diff
@@ -50,5 +66,26 @@ def getARs(self, analysis_state):
         bc = getToolByName(self, 'bika_catalog')
         return [p.getObject() for p in bc(portal_type='AnalysisRequest', getDoctorUID=self.UID())]
 
+    def get_clients(self):
+        # Only show clients to which we have Manage AR rights.
```

Comment:
Instead of a comment we could put the documentation of the function in a docstring?